src/pages/Xudt/HolderAllocation.tsx

```diff
@@ -62,7 +64,13 @@ const HolderAllocation = ({
                   .map(amount => (
                     <tr>
                       <td>
-                        <EllipsisMiddle>{matchScript(amount.codeHash)?.tag ?? amount.codeHash}</EllipsisMiddle>
+                        {matchScript(amount.codeHash) ? (
+                          <EllipsisMiddle>{matchScript(amount.codeHash)?.tag ?? amount.codeHash}</EllipsisMiddle>
+                        ) : (
+                          <Link className={styles.scriptLink} to={`/script/${amount.codeHash}/${amount.hashType}`}>
+                            <EllipsisMiddle>{matchScript(amount.codeHash)?.tag ?? amount.codeHash}</EllipsisMiddle>
+                          </Link>
+                        )}
```

Comment:
`matchScript` is called 3 times so it can be assigned to an variable.

And in the `else branch`, `matchScript(amount.codeHash)` is `falsy`, the `nullish + coalesce` is unnecessary.

Reply:
Any update on this conversation?